src/app/components/kalkulator-bmi/kalkulator-bmi.component.ts

```diff
@@ -0,0 +1,55 @@
+import { Component } from "@angular/core";
+
+@Component({
+    selector: 'app-kalkulator-bmi',
+    templateUrl: './kalkulator-bmi.component.html',
+    styleUrls: ['kalkulator-bmi.component.scss']
+})
+export class KalkulatorBmiComponent {
+    public showResults: boolean = false;
+    public resultText: string = "";
+    public resultNumber: string = "";
+    public weight: number = 0;
+    public height: number = 0;
+    public bmi: number = 0;
```

Comment:
Czy tej zmiennej używasz w widoku (pliku html)? Jeśli nie, to powinna być private a nie public

Reply:
Zmienna bmi?